src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.tsx

```diff
@@ -122,11 +128,6 @@ type Props = {
    */
   showRelative?: boolean;
 
-  /**
-   * Allow user to clear the time range selection
-   */
-  allowClearTimeRange?: boolean;
```

Comment:
Previously only used by "Performance" - is replaced with `defaultSelection`. This also fixes inconsistency with default dates in "Performance".  cc @dashed

Reply:
Ok.